src/utils.js

```diff
@@ -279,7 +279,7 @@ export function getDefaultFormState(
     // Override schema defaults with form data.
     return mergeObjects(defaults, formData);
   }
-  if (formData === 0 || formData === false) {
+  if (formData === 0 || formData === false || formData === "") {
```

Comment:
@Kexin-Li thanks for your change! Can you add a test for this?

It can be modeled off of this -- https://github.com/rjsf-team/react-jsonschema-form/blob/18b2ec42c9cfc4aad2cf6996342c8edde979ee68/test/StringField_test.js#L76 -- basically, you want to test that when a textbox is cleared, its value should also be "" instead of the default value.

Reply:
Sure!